content/docs/start/index.md

```diff
@@ -53,12 +59,13 @@ next few sections:
 
 - [**Data access**](/doc/start/data-access) shows how to use data artifacts from
   outside of the project and how to import data artifacts from another DVC
-  project. This can help to download a specific version of an ML model to a
-  deployment server or import a model to another project.
+  project. This can help to download a specific version of a machine learning
+  model to a deployment server or import a model to another project.
```

Comment:
Why don't we change this in the previous bullet instead, which is the first instance of "ML"?

Reply:
OK, moved full term to previous bullet and reverted this.

Reply:
Thanks